package/MDAnalysis/topology/c_GROParser.pyx

```diff
@@ -86,6 +87,12 @@ class GROParser(TopologyReaderBase):
         # resid, resname, name, index, (x,y,z)
         cdef int n_atoms;
         cdef int i;
+        cdef str line
+        cdef int count = 0
+        cdef np.ndarray[np.int32_t] resids
```

Comment:
Defining arrays with numpy  C interface is causing the build to fail probably, though the build is successful locally. Any ideas how to deal with this?